pkg/provider/github/status.go

```diff
@@ -124,16 +140,40 @@ func (v *Provider) getOrUpdateCheckRunStatus(ctx context.Context, runevent *info
 	return err
 }
 
+func (v *Provider) updatePipelineRunWithCheckRunID(ctx context.Context, tekton versioned.Interface, pr *v1beta1.PipelineRun, checkRunID *int64) error {
+	if pr == nil {
+		return nil
+	}
+	maxRun := 10
+	for i := 0; i < maxRun; i++ {
+		pr, err := tekton.TektonV1beta1().PipelineRuns(pr.GetNamespace()).Get(ctx, pr.GetName(), v1.GetOptions{})
+		if err != nil {
+			return err
```

Comment:
do we need to skip not found, in case it was deleted in between ?

Reply:
pr was just created, not sure if required to check not found🤔